nilrt_snac/_Configs/_CryptsetupConfig.py

```diff
@@ -0,0 +1,23 @@
+import argparse
+
+from nilrt_snac._Configs._BaseConfig import _BaseConfig
+
+from nilrt_snac import logger
+from nilrt_snac.OpkgHelper import opkg_helper
+
+
+class _CryptsetupConfig(_BaseConfig):
+    def __init__(self):
+        self._opkg_helper = opkg_helper
+
+    def configure(self, args: argparse.Namespace) -> None:
+        logger.info("Configuring cryptsetup...")
+
+        self._opkg_helper.install("cryptsetup")
```

Comment:
No dry run?

Reply:
OpkgHelper knows what the dry_run flag's value is and only install/removed when dry_run is False.